[2024-09-10T07:17:18.049+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T07:17:18.076+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V2.second_task scheduled__2021-10-13T00:00:00+00:00 [queued]>
[2024-09-10T07:17:18.096+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V2.second_task scheduled__2021-10-13T00:00:00+00:00 [queued]>
[2024-09-10T07:17:18.098+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-10T07:17:18.142+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): second_task> on 2021-10-13 00:00:00+00:00
[2024-09-10T07:17:18.160+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V2', 'second_task', 'scheduled__2021-10-13T00:00:00+00:00', '--job-id', '1007', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpm93vrk5c']
[2024-09-10T07:17:18.166+0000] {standard_task_runner.py:105} INFO - Job 1007: Subtask second_task
[2024-09-10T07:17:18.161+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=5629) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-10T07:17:18.178+0000] {standard_task_runner.py:72} INFO - Started process 5639 to run task
[2024-09-10T07:17:18.272+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_V2.second_task scheduled__2021-10-13T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-10T07:17:18.577+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_V2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2021-10-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-10-13T00:00:00+00:00'
[2024-09-10T07:17:18.586+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T07:17:18.647+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-10T07:17:18.650+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is the second task!']
[2024-09-10T07:17:18.675+0000] {subprocess.py:86} INFO - Output:
[2024-09-10T07:17:18.684+0000] {subprocess.py:93} INFO - hey, this is the second task!
[2024-09-10T07:17:18.706+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-10T07:17:18.803+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T07:17:18.817+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V2, task_id=second_task, run_id=scheduled__2021-10-13T00:00:00+00:00, execution_date=20211013T000000, start_date=20240910T071718, end_date=20240910T071718
[2024-09-10T07:17:19.066+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T07:17:19.144+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-10T07:17:19.179+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
